src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/LazyPInvokeHelpers.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Maybe just put it into `src\System.Private.CoreLib\src\Internal\Runtime\CompilerHelpers\InteropHelpers.cs`. I expect that all interop helpers will eventually disappear or move someplace else - once we get the full MCG-based one.